functional/cypress-tests/cypress/integration/widgets.js

```diff
@@ -1,13 +1,13 @@
-import Widgets from '../fixtures/Widgets';
-import { fillDynamicTab } from '../support/widgetDynamicTab';
-import { validateWidgetConfig } from '../support/widgetAssertions';
+import Widgets from "../fixtures/Widgets";
+import { fillDynamicTab } from "../support/widgetDynamicTab";
+import { validateWidgetConfig } from "../support/widgetAssertions";
 let example = Widgets.example;
-let dashboardName = 'Welcome to Cogboard';
+let dashboardName = "Welcome to Cogboard";

```

Comment:
For the changes to quotes - are you sure you don't have something installed on your machine that changes this? Any sane linter would leave as it was before, and I don't see it done this way in other people's PRs.

Reply:
Please use single quotes. I know that this part of the project is outside of the webapp part (what is not good IMO, because we have linter and prettier setup for js), but we have to be consistent with other javascript files in the project.